tools/src/main/java/org/apache/kafka/trogdor/workload/ConsumeBenchWorker.java

```diff
@@ -77,9 +85,12 @@ public void start(Platform platform, WorkerStatusTracker status,
             throw new IllegalStateException("ConsumeBenchWorker is already running.");
         }
         log.info("{}: Activating ConsumeBenchWorker with {}", id, spec);
+        this.statusUpdater = new StatusUpdater();
         this.executor = Executors.newScheduledThreadPool(
-            2, ThreadUtils.createThreadFactory("ConsumeBenchWorkerThread%d", false));
-        this.status = status;
+            spec.consumerCount() + 2, // 1 thread for all the ConsumeStatusUpdater and 1 for the StatusUpdater
+            ThreadUtils.createThreadFactory("ConsumeBenchWorkerThread%d", false));
+        this.statusUpdaterFuture = executor.scheduleAtFixedRate(this.statusUpdater, 2, 1, TimeUnit.MINUTES);
```

Comment:
Hmm.  Why change the initial delay from 1 minute to 2?

Reply:
Whoops, I initially thought this was seconds. My reasoning was that since there are more threads spawned and a rebalance happening, it would make sense to wait a bit more before exposing the status.
Anyway, reverting this back to 1 minute